src/server/api/routers/post.ts

```diff
@@ -46,7 +48,23 @@ export const postRouter = createTRPCRouter({
             // query: z.function().args().returns(z.string())
         }))
         .query(async ({input}) => {
-            const queryURI = encodeURI(input.query);
+            console.log("SEARCH: " + input.query)
+
+            const openai = new OpenAI({
+                apiKey: env.OPENAI_API_SECRET
+            })
+
+            const chatCompletion = await openai.chat.completions.create({
+                model: "gpt-4-turbo-preview",
+                messages: [{"role": "system", "content": `
+                You will be given text. Find the opposite text to the prompt.
+
+                For example, If the input is 'Hot locations near me', response should be 'Cold locations far away from me'
```

Comment:
Could we include more examples that cover more possible input cases? If I search "GitHub" what will it give me? If I search "What time is it" what happens?

It may also be worth giving the input as JSON and requesting JSON as output, as that prevents GPT from saying "Sure! Here is the opposite of ..."